packages/server/modules/core/graph/resolvers/streams.ts

```diff
@@ -1,8 +1,5 @@
-'use strict'
-const { UserInputError } = require('apollo-server-express')
-const { withFilter } = require('graphql-subscriptions')
-
-const {
+import { UserInputError } from 'apollo-server-express'
```

Comment:
I didn't really want to, but had to refactor the streams resolvers to `.ts` cause there was a lot of Invites related code here and without TS linting there was no way to be sure that my extensive invite refactors hadn't broke anything